PC/winreg.c

```diff
@@ -964,23 +969,36 @@ If the function fails, an OSError exception is raised.
 
 static HKEY
 winreg_CreateKeyEx_impl(PyObject *module, HKEY key, const wchar_t *sub_key,
-                        int reserved, REGSAM access)
-/*[clinic end generated code: output=51b53e38d5e00d4b input=42c2b03f98406b66]*/
+                        int reserved, REGSAM access, int options,
+                        int create_only)
+/*[clinic end generated code: output=10c0a5f7beea07e3 input=23d740b8cd7fb0df]*/
 {
     HKEY retKey;
     long rc;
+    DWORD disposition;
 
-    if (PySys_Audit("winreg.CreateKey", "nun",
+    if (PySys_Audit("winreg.CreateKey", "nunii",
                     (Py_ssize_t)key, sub_key,
-                    (Py_ssize_t)access) < 0) {
+                    (Py_ssize_t)access, options, create_only) < 0) {
         return NULL;
     }
```

Comment:
We can't change the schema here. If we need to audit those values (I'm not sure we do), then create a second event `winreg.CreateKey/2` and include everything in that.